NSS-Day-Cohort-53/kennels-econolines
# Employee details

#### Changes Made
1. Modified file `employee.js` to include `animal count and location name` 
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout sh-employeeDetails
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Click on an employee to bring up their detail card, when the employee card is displayed a count of animals cared for and the name of the location should be displayed
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
